libsolidity/interface/StandardCompiler.cpp

```diff
@@ -200,6 +201,7 @@ bool isBinaryRequested(Json::Value const& _outputSelection)
 	static vector<string> const outputsThatRequireBinaries{
 		"*",
 		"ir", "irOptimized",
+		"ewasm.wast", "ewasm.wasm",
```

Comment:
What about just `wast`? It is considered in the other function above

Reply:
yep, I'll add it.